lib/helpers.js

```diff
@@ -40,6 +40,45 @@ var Helpers = function (ngFaker) {
 
     return state;
   };
+
+  /**
+   * slugifies string
+   *
+   * @method ngFaker.helpers.slugify
+   * @param {string} string
+   */
+  self.slugify = function (string) {
+    string = string || '';
+    return string.replace(/ /g, '-').replace(/[^\w.-]+/g, '');
+  };
+
+  /**
+   * pastYear
+   *
+   * @method ngFaker.helpers.pastYear
+   * @param {number} years
+   * @param {date} reference date
+   *
+   */
+  self.pastYear = function (years, refDate) {
+    /**
+     * If we ever decide to generate dates we
+     * can move this function into a date class.
+     * For now, this can remain as a helper
+     */
+    var date = (refDate) ? new Date(Date.parse(refDate)) : new Date();
```

Comment:
What if the format of the refDate isn't what we want?
If I enter a random string then the value of date becomes `invalid date` which will have an effect on the result of the method.

You might need to had a check to ensure the type of `refDate` is date.

Reply:
Good spot @BolajiOlajide I'll fix that